pkg/controller/controller.go

```diff
@@ -106,9 +107,9 @@ func computeChangeList(current []model.Config, desired []model.Config, errHandle
 		_, exists := desiredMap[key]
 		if !exists {
 			item := processor.Item{
-				Operation:    model.EventDelete,
-				Resource:     currConfig,
-				ErrorHandler: errHandler,
+				Operation:       model.EventDelete,
+				Resource:        currConfig,
+				CallbackHandler: errHandler,
```

Comment:
Can we rename all these `errHandler` to `cbHandler` or something along those lines everywhere?

Reply:
oops, my bad. I thought I had updated it.